desktop/cmp/rest/RestGridModel.js

```diff
@@ -118,26 +165,20 @@ export class RestGridModel {
                 {
                     text: 'Add',
                     icon: Icon.add(),
-                    action: () => this.addRecord()
+                    actionFn: () => this.addRecord()
                 },
                 {
-                    text: 'Edit',
-                    icon: Icon.edit(),
-                    action: (item, record) => this.editRecord(record),
-                    recordsRequired: 1
+                    ...this.editAction
```

Comment:
Do these need to be cloned here?   also think Add should be defined like the other two for symmetry.